cypress.config.ts

```diff
@@ -1,25 +1,26 @@
-import { defineConfig } from "cypress";
+import {defineConfig} from "cypress";
 
 export default defineConfig({
-  chromeWebSecurity: false,
-  video: false,
+    chromeWebSecurity: false,
+    video: false,
 
-  e2e: {
-    // We've imported your old cypress plugins here.
-    // You may want to clean this up later by importing these.
-    setupNodeEvents(on, config) {
-      // eslint-disable-next-line @typescript-eslint/no-var-requires
-      return require("./cypress/plugins/index.js")(on, config);
+    e2e: {
+        // We've imported your old cypress plugins here.
+        // You may want to clean this up later by importing these.
+        setupNodeEvents(on, config) {
+            // eslint-disable-next-line @typescript-eslint/no-var-requires
+            return require("./cypress/plugins/index.js")(on, config);
+        },
+        baseUrl: "http://localhost:3000",
+        viewportHeight: 1500,
+        viewportWidth: 2000,
+        experimentalRunAllSpecs: true
```

Comment:
denne er lagt til for å gjøre det mulig å teste flere specs i en run, påvirker ikke kjøring, gjelder kun for testing lokalt